README.md

```diff
@@ -158,6 +158,22 @@ This should get you a starter Islandora site with:
 * RDF and JSON-LD mappings for miscellaneous entities to support storage in
 Fedora, Triplestore indexing and client requests.
 
+### Private file system
+
+* In Drupal container of isle-dc, create private file system directory
```

Comment:
The Starter Site is not specific to ISLE-dc.

Reply:
Will test it with playbook and isle-site-template as well.